src/systemd/sd-dhcp-client.h

```diff
@@ -222,6 +222,21 @@ enum {
         SD_DHCP_RELAY_AGENT_REMOTE_ID              = 2
 };
 
+enum DHCPState {
```

Comment:
Hm. Sorry, but after-reconsidering about exporting the state in `sd-` header, this (and also the one for DHCPv6) should be internal, at least now.

Reply:
No problem. I put these internal interfaces that are intended to be used by the dhcp client within the `dhcp-client-internal.h` and `dhcp6-client-internal.h` headers. Let me know if that isn't the right approach.